gulpfile.js

```diff
@@ -84,6 +84,11 @@ gulp.task('ps1tests', ['compileTests'], function (cb) {
         .pipe(gulp.dest(_testRoot));
 });
 
+gulp.task('copyTestRes', ['compileTests'], function (cb) {
```

Comment:
Naming the folder `data` instead of `resources` might be more clear. The word "resources" is already overloaded.

Also instead of adding a new gulp task, it looks like you could do this within the existing `testResources` task. The body of that task is currently empty.